src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,11 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
```

Comment:
from check-list:
Don’t push jsp file, jsp dependencies, hello controller and InternalResourceViewResolver resolver() bean to PR.